Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -129,23 +180,39 @@ extension BRBViewController: BRBConnectionDelegate {
         }
     }
     
-    func loginFailed(with error: String) {
-        loginViewController.errorDescription = error
-        loginViewController.setShowErrorMessage(true, animated: true)
-        loginViewController.isLoading = false
+    func failedToGetAccount(with error: String) {
+        isLoading = false
+        if loggedIn {
+            showErrorAlert(error: error)
+        } else {
+            loginViewController.errorDescription = error
+            loginViewController.setShowErrorMessage(true, animated: true)
+            loginViewController.isLoading = false
+        }
     }
     
 }
 
 extension BRBViewController: AboutTableViewControllerDelegate {
-    
+
     func aboutTableViewControllerDidLogoutUser(_ stvc: AboutTableViewController) {
         setState(.login)
-        
-        connectionHandler = BRBConnectionHandler()
-        connectionHandler.delegate = self
-        
+        UserDefaults.standard.set(nil, forKey: "BRBAccount")
+        navigationItem.title = "Meal Plan"
+        accountManager = BRBAccountManager()
+        accountManager.delegate = self
+        accountManager.removeSavedLoginInfo()
```

Comment:
Do we need to recreate the account manager here? If this is used to reset the connection handler, then maybe adding a method on `BRBAccountManager` called `resetConnection` might be better.